Cargo.lock

```diff
@@ -1805,6 +1865,20 @@ name = "nan-preserving-float"
 version = "0.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
+name = "native-tls"
+version = "0.1.5"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+dependencies = [
+ "lazy_static 0.2.11 (registry+https://github.com/rust-lang/crates.io-index)",
+ "libc 0.2.36 (registry+https://github.com/rust-lang/crates.io-index)",
+ "openssl 0.9.24 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
This dependency will require ```OpenSSL``` and is by default dynamically linked! For a while ago we got rid of ```OpenSSL``` thus, we need to update all docker images and similar!

/cc @kirushik 
Do you have any thoughts about this regarding the binary size etc?

Reply:
Yeah, I agree it really can be a blocker here. I'll try to find out what can be done here. It's interesting that `hyper` on it own does not require OpenSSL to connect over HTTPS. But `hyper-proxy` seems to require it somehow. Strange.